src/middleware/SessionMocker.ts

```diff
@@ -0,0 +1,16 @@
+import {NextFunction, Response, Request} from "express";
+
+
+export function mocker (req:Request, res:Response, next:NextFunction): void {
```

Comment:
I don't think the 'req' parameter is used

Reply:
if I remove req, app.use in configLoader gets upset

Reply:
just seen this!